core/src/main/java/org/hisp/dhis/android/core/domain/metadata/MetadataCall.kt

```diff
@@ -96,80 +99,71 @@ internal class MetadataCall @Inject constructor(
 ) {
 
     companion object {
-        const val CALLS_COUNT = 12
+        const val CALLS_COUNT = 15
```

Comment:
why 15?

Reply:
There are 15 calls, it was outdated before. This is just a hint for the app in case it wants to show a progress bar, it does not affect anything else.